### PR TITLE
PR for #70

### LINKED ISSUE #70: Add jump to declaration for add_shortcode
#### Issue type

- [ ] Bug
- [x] Enhancement

#### Overview description
It is possible to navigate to declaration on ctrl + click for the method name inside of add_filter and add_action. Unfortunately, this is not possible for add_shortcut and maybe other add_* methods who uses methodnames in method call.

#### Steps to reproduce

1. Write add_shortcut('your_shortcode_name', 'your_shortcode_callback'); in your functions php or plugin.
2. Ctrl + hover string 'your_shortcode_callback'.

#### Actual results
See no link to click and jump to the method declaration

#### Expected results
See link to click and jump to the method declaration as for add_filter and add_action.

#### Your environment (NetBeans and plugin version, e.t.c.)
Windows 10 64Bit, NetBeans 8.2 PHP WordPress Blog/CMS 0.9.5

### PR DESCRIPTION
**Pull request by Chris2011**
This PR will fix #70 and adding go to declaration for strings inside add_shortcode, register_activation_hook and register_deactivation_hook.

### CODE REVIEW

**Comment by junichi11** [comment]
Don't choose `master` as a base branch. Instead, `nb82dev`.

File: manifest.mf [hunk 1]
```diff
@@ -2,5 +2,5 @@ Manifest-Version: 1.0
 OpenIDE-Module: org.netbeans.modules.php.wordpress
 OpenIDE-Module-Layer: org/netbeans/modules/php/wordpress/resources/layer.xml
 OpenIDE-Module-Localizing-Bundle: org/netbeans/modules/php/wordpress/Bundle.properties
-OpenIDE-Module-Specification-Version: 0.9.5
+OpenIDE-Module-Specification-Version: 0.9.6
```
Comment (junichi11) [inline manifest.mf]: Don't change this.

Comment (Chris2011) [reply manifest.mf]: Should I reset/revert it or make a new commit?

Comment (junichi11) [reply manifest.mf]: Choose the way you can.

**Comment by Chris2011** [comment]
Done.

**Pull request merged**